control/server.py

```diff
@@ -41,6 +42,17 @@ def callable():
 
     return callable
 
+def wait_for_condition(condition_method: Callable[[], bool], error_message=None, timeout=None, interval=1):
```

Comment:
Now I see the type annotations... We should [enable mypy or any other static type checker](https://github.com/ceph/ceph-nvmeof/issues/188).